src/test/java/uk/gov/justice/hmpps/prison/repository/BookingRepositoryTest.java

```diff
@@ -166,7 +166,7 @@ public void testGetLatestBookingByBookingIdHavingInactiveBooking() {
 
         assertThat(summary.getOffenderNo()).isEqualTo("Z0020ZZ");
         assertThat(summary.getFirstName()).isEqualTo("BURT");
-        assertThat(summary.getMiddleNames()).isNull();
+        assertThat(summary.getMiddleNames()).isNullOrEmpty();
```

Comment:
This is a consequence of the HSQLDB change. On NOMIS this does return null as expected, but on HSQLDB it returns empty.